string label
point center_point


